.travis.yml

```diff
@@ -29,6 +29,7 @@ before_script:
   - chmod +x ./install-byond.sh
   - ./install-byond.sh
 script:
+ - (num=`grep -E '\\\\(red|blue|green|black|b|i[^mc])' **/*.dm | wc -l`; echo "$num escapes (expecting ${MACRO_COUNT} or less)"; [ $num -le ${MACRO_COUNT} ])
```

Comment:
I blatantly ripped this from Bay. Could someone that's more familiar with Travis/grep ensure this works?